<launch>
    <!-- Start the turtlesim node -->
    <node name="turtlesim" pkg="turtlesim" type="turtlesim_node" output="screen" />

    <!-- Delay for a moment to allow turtlesim to initialize the default turtle -->
    <node name="remove_default_turtle" pkg="rosservice" type="rosservice" args="call /kill 'turtle1'" output="screen" required="true">
        <param name="respawn_delay" type="double" value="1.0" />
    </node>
</launch>
